cmd/state-installer/installer_lin_mac.go

```diff
@@ -0,0 +1,34 @@
+//go:build !windows
+// +build !windows
+
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/fileutils"
+)
+
+func (i *Installer) cleanInstallPath() error {
```

Comment:
We should make sure here that we don't need to worry about the additional application files that are created on macOS.

Reply:
The system (launcher) files are reinstalled with every update. So we rewrite the install files including any symlinks so this shouldn't be an issue on macOS.